packages/react/src/Card/Card.module.css

```diff
@@ -111,21 +111,12 @@
 }
 
 .Card__icon {
-  margin-bottom: var(--base-size-16);
+  margin-bottom: var(--base-size-24);
```

Comment:
Was this extra spacing an intended design change? It's increasing the height of the card

Reply:
Yes intentional. Called this out in the PR description, but this aligns the component with the Figma

Reply:
It seems we've had this discrepancy between Figma and React for some time. The `24px` spacing has always been present in the Figma implementation (see interface guidelines images, for example) and was not introduced with the new Icon component update. 

I can confirm that 24 is the intended spacing here.